ndcube/tests/test_global_coords.py

```diff
@@ -0,0 +1,25 @@
+import astropy.units as u
+from ndcube.global_coords import GlobalCoords
+
+gc = GlobalCoords()  # Initialze an empty GlobalCoords instance
+
+# Init
+coord1 = 1 * u.m
+coord2 = 2 * u.s
```

Comment:
You shouldn't have things defined in the root of the file, they should either be in tests or in fixtures.

Reply:
Okay, I see

Reply:
Moved to within the test proper